contents/blog/feature-flag-best-practices.md

```diff
@@ -0,0 +1,198 @@
+---
+date: 2023-01-03
+title: 8 best practices for feature flags and toggles
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: ["ian-vanagas"]
+featuredImage: ../images/blog/green-blog-image.jpg
+featuredImageType: full
+categories: ["Guides", "Using PostHog"]
+---
+
+Feature flags (also known as feature toggles) enable teams to make changes to their applications quickly and safely. They check whether a piece of code should be active or not and a simple UI to create and manage those checks. This enables fast feature testing, rollout, and rollback.
+
+There are several best practices for implementing feature flags correctly. This post goes over 8 of them, from using local evaluation to deciding when to remove feature flags.
```

Comment:
```suggestion
Feature flags, aka feature toggles, are awesome. Want to test a new feature on your beta test group? Use a feature flag. Want to test multiple variants of a new UX? Use a feature flag? Need a kill switch to prevent performance problems? Yup, feature flag. We could go on, but suffice it to say the benefits of feature flags are numerous.
Here we're exploring feature flag best practices, such as deciding when to remove them and naming conventions. Some are specific to using [feature flags in PostHog](https://github.com/product/feature-flags), but the principles apply no matter what [feature flag tool](https://github.com/blog/best-open-source-feature-flag-tools) you use.
```

Reply:
Bit of a punch up on the intro. Feel free to tweak.